### PR TITLE
Update center_write.md

### PR DESCRIPTION
**Pull request by macks2008**
rephrased and elaborated the explanation of the suggested code. Also a slight refactor to extract the new X value (to make it easier to describe the steps). I don't THINK I broke the example, but I also didn't test it to be sure.

This all started because I noticed the apostrophe after the word "object" was playing hide and seek with you. Funny how that happens sometimes...

### CODE REVIEW

File: faqs/center_write.md [hunk 1]
```diff
@@ -7,13 +7,16 @@ There is no builtin function to write text center-aligned in ComputerCraft, but
...
     term.write(text)
 end
 ```
 After declaring this function, you can use it like this:
 ```lua
 centerWrite("Hello world!")
 ```
-This code gets the current `term` objects size. Now, to get a starting point for writing text, it divides the `term` width by 2. It then subtracts half of the text length, and what results is the x value to start writing the text from to appear 'centered'.
+This code starts by getting the current `term` object's size. Then it calculates the horizontal starting position for writing text by dividing the terminal width by 2, and subtracting half of the text length (the math.ceil it is to make sure the result is an integer). 
+Finally, it sets the text cursor  to this x position, and the cursor's current y position.
```
Comment (SquidDev) [inline faqs/center_write.md:20]:
```suggestion
Finally, it moves the cursor to this x position and the current y position.
```

File: faqs/center_write.md [hunk 2]
```diff
@@ -7,13 +7,16 @@ There is no builtin function to write text center-aligned in ComputerCraft, but
 local function centerWrite(text)
     local width, height = term.getSize() -- Get terminal size
     local x, y = term.getCursorPos() -- Get current cursor position
-    term.setCursorPos(math.ceil((width / 2) - (text:len() / 2)), y)
+    local newX = math.ceil((width / 2) - (text:len() / 2))
+    term.setCursorPos(newX, y)
```
Comment (SquidDev) [inline faqs/center_write.md:11]:
```suggestion
    local new_x = math.ceil((width / 2) - (text:len() / 2))
    term.setCursorPos(new_y, y)
```

**Review by SquidDev — CHANGES_REQUESTED** [review]

**Comment by SquidDev** [comment]
The phrasing of the whole FAQ is pretty clunky, but this is a definite improvement. Thank you!

File: faqs/center_write.md [hunk 3]
```diff
@@ -7,13 +7,16 @@ There is no builtin function to write text center-aligned in ComputerCraft, but
 local function centerWrite(text)
     local width, height = term.getSize() -- Get terminal size
     local x, y = term.getCursorPos() -- Get current cursor position
-    term.setCursorPos(math.ceil((width / 2) - (text:len() / 2)), y)
+    local new_x = math.ceil((width / 2) - (text:len() / 2))
+    term.setCursorPos(new_y, y)
```
Comment (Wojbie) [inline faqs/center_write.md:11]: Typo

**Pull request merged**